internal/dataplane/translator/subtranslator/ingress_atc.go

```diff
@@ -38,7 +35,13 @@ func (m *ingressTranslationMeta) translateIntoKongExpressionRoute() *kongstate.R
 		ingressHost = strings.ReplaceAll(ingressHost, "*", "_")
 	}
```

Comment:
nit: Lines 35:39 can be replaced with the below

```go
// '_' is not allowed in a host, so use '_' to replace a possible occurrence of  '*' since '*' is not allowed in Kong.
ingressHost := strings.ReplaceAll(ingressHost, "*", "_")
```